atlp-rwanda/e-comm-team-brogrammers-fn
# #184948165 a seller delete item and update availability

#### What does this PR do?

this PR is for deleting item and update availability in seller collection

#### Description of Task to be completed?

a seller should be able to delete a specific item from his/her collection but also a seller should be able to update the product availability 

#### How should this be manually tested?

-first log in if you are a seller or an admin
- go to route  `collection/manageProducts` to be able to see all your products
-  click on the three dots on  right bottom corner to get buttons if you want to delete the product you can delete it and if to disable or enable product click on respective button

#### Any background context you want to provide?

n/a

#### What are the relevant pivotal tracker/Trello stories?

https://www.pivotaltracker.com/story/show/184948165
#### Screenshots (if appropriate)

<img width="777" alt="Screenshot 2023-05-12 at 11 45 04" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/99663528/fbab80bb-f54e-4342-8a76-00e243877576">

<img width="573" alt="Screenshot 2023-05-12 at 11 42 56" src="https://github.com/atlp-rwanda/e-comm-team-brogrammers-fn/assets/99663528/7ae45fc2-0738-45be-aeab-a6e8cb0f3c01">


#### Questions:
n/a
